5 view one article

WiccanWolf opened the pull request.

nc-mohammed-dastgir reviewed (commented):
This is a good start, but please do stick to the task features and try not to branch out too much as most of this pr should actually be multiple pr's and some things like commented out code should not be included and push up to github if it is not needed.

Please do also have a read of my below comments:

WiccanWolf commented:
Oh @nc-mohammed-dastgir my apologies I added the context file to my .gitignore so I'm confused as to why it was even pushed?? I've removed the context files from my code altogether after speaking with Mitch and future PR should reflect this 👍
